frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -382,16 +382,20 @@ export const useApplicationsInProgressQuery = ({
   page = 0,
   take = 10,
   searchString = "",
-  sorting = [],
+  orderBy = [],
+}: {
+  page?: number;
+  take?: number;
+  searchString?: string;
+  orderBy?: SortingConfig[];
 }) => {
-  const applicationsInProgressQuery = useQuery({
-    queryKey: ["applicationInProgress"],
+  return useQuery({
+    queryKey: ["applicationsInProgress", page],
```

Comment:
add the `page` to the `queryKey` for pagination

Reply:
what about when the page size changes, or the sorting params change?

Reply:
updated the `queryKey` hash with the `take` param to accomodate the page size changing, but do we need to handle a sorting params change at this time since the table doesn't provide a sort option..?

Reply:
There is a ticket to allow sorting. We have currently disabled it in the frontend as there was some confusion around the requirements.

Reply:
ok - I may as well add a sortingKey to this hash then just in case